submissions/jeffersonMormah/htmlForm/signup.html

```diff
@@ -56,28 +53,22 @@ <h1><em>Register your account</em></h1>
                         <li><a href = "#" >Terms and Conditions</a></li>
                 </ul>
         </div>
-
         <div class="footer2">
                 <label for="newsletter">Sign up for our Newsletter</label>
                 <br>
                 <input type="email" name="newsletter" id="email" placeholder="Enter your email address">
-                <button type="submit" name="submit_form" id="submit_form">SUBMIT</button>
+                <button type="submit" name="submit-button" id="submit-button">SUBMIT</button>
         </div>
-
-        
         <div class="footer3">
                 <h5>CONNECT WITH US</h5>
                 <ul >
                     <li><a href="#"><img src="../introductionToHtml/images/facebook.jpg" alt="Facebook"></a></li>
                     <li><a href="#"><img src="../introductionToHtml/images/twitter.jpg" alt="Twitter"></a></li>
-                    <li><a href="#"><img src="../introductionToHtml/images/instagram.jpg" alt="Instagram"></a></li>
-                    
+                    <li><a href="#"><img src="../introductionToHtml/images/instagram.jpg" alt="Instagram"></a></li> 
                 </ul>
             </div>
         <br/><br/>
             <p>Copyright 2020 Ebekel chemicals All rights reserved</p>
-        
-      
         </footer>
```

Comment:
I can't find the opening tag of `<footer>`

Reply:
@Jeffbuchi